CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+### Added
```

Comment:
I like the use of semantic versioning, but i don't think each release should have 3 different categories, i think that will become cumbersome